oak_containers/proto/launcher.proto

```diff
@@ -27,11 +27,20 @@ message GetImageResponse {
   bytes image_chunk = 1;
 }
 
+message GetContainerConfigResponse {
+  // Arbitrary config that the container can retrieve from the orchestrator.
+  // Included in the attestation measurements conducted by the orchestrator.
+  bytes config = 1;
+}
```

Comment:
Sending this as a simple message, since I'd think it's safe to assume that the config is below 4MiB cc @tiziano88